.github/workflows/ci.yml

```diff
@@ -18,6 +18,8 @@ jobs:
           - stm32f302xc
           - stm32f302xd
           - stm32f302xe
+          - stm32f302x6
+          - stm32f302x8
```

Comment:
There is no Travis involved, we are using Github Actions now :)
Here is the relevant documentation for matrices: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy

Basically, it runs the job for each combination of all specified features. In our case it means, every MCU is checked with the "rt" feature. The "include" field is special and allows you to add new entries to the matrix or overwrite existing ones. We use that to specify that the Check for the stm32f303xc MCU should use the features "rt,stm32-usbd" instead of "rt". But all other MCUs are still tested. You can actually see that in the names of the checks executed for a PR.

Not sure which note in the README you mean? There is one in the Cargo.toml, that already mentions the ci.yml file:
```toml
# Any Changes here should be mirrored in README.md, src/lib.rs, and
# .github/workflows/ci.yml.
```

Reply:
> There is no Travis involved, we are using Github Actions now :)
> Here is the relevant documentation for matrices: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategy
>
> Basically, it runs the job for each combination of all specified features. In our case it means, every MCU is checked with the "rt" feature. The "include" field is special and allows you to add new entries to the matrix or overwrite existing ones. We use that to specify that the Check for the stm32f303xc MCU should use the features "rt,stm32-usbd" instead of "rt". But all other MCUs are still tested. You can actually see that in the names of the checks executed for a PR.

Thanks for taking the time to explain this. I like it.
> 
> Not sure which note in the README you mean? There is one in the Cargo.toml, that already mentions the ci.yml file:
> 
> ```toml
> # Any Changes here should be mirrored in README.md, src/lib.rs, and
> # .github/workflows/ci.yml.
> ```
In the root README there is a comment on line 50 I was talking about:
`[comment]: # (Any changes here should be mirrored in src/lib.rs)`
I'm not sure if adding the instructions to all places where one can possibly think about makes sense or if it makes more sense to add a central note in the README on "contributing" and reference that.

Reply:
Ha, good catch. Somehow I totally missed this comment in the README.

> I'm not sure if adding the instructions to all places where one can possibly think about makes sense or if it makes more sense to add a central note in the README on "contributing" and reference that.

Probably a good idea. I'm also not a huge fan of having to repeat all the features so often. I think it would be better to just tell the user to pick one feature from the Cargo.toml. But that discussion doesn't really fit here I guess. 

How about simply removing this comment in the README? IMO having it only in the Cargo.toml is enough, because if you want to add a new feature you will obviously do it there first. Nobody (I hope) will think that just adding a new line to the README will magically make a new feature appear in the code. Plus the README is targeted to users anyway, not developers.